pygsti/tools/matrixtools.py

```diff
@@ -904,9 +954,13 @@ def norm1(m):
     -------
     numpy.ndarray
     """
-    return float(_np.real(_np.trace(_sqrtm(_np.dot(m.conj().T, m)))))
+    s = _spl.svdvals(m)
+    nrm = _np.sum(s)
+    return nrm
 
 
+# Riley note: I'd like to rewrite this, but I don't want to mess with reproducibility
```

Comment:
Lol what reproducibility? We aren't seeding the RNG here so this is essentially impossible to reproduce without being very careful about calls to np.random in all scopes...
I'm generally in favor of a new implementation, or at least one where we can seed it such that we make reproducing it possible. Doesn't have to be in this PR, but just a future to-do.

Reply:
Fair point! I have a slight preference to defer these discussions till a new PR.